dashboard/app/models/pd/application/application_base.rb

```diff
@@ -221,6 +221,10 @@ def applicant_name
       "#{sanitize_form_data_hash[:first_name]} #{sanitize_form_data_hash[:last_name]}"
     end
 
+    def response_score_hash
+      JSON.parse(response_scores || '{}').symbolize_keys
+    end
```

Comment:
nit: should this be `response_scores_hash` to match the plurality of the underlying attribute?